pkg/rke2/rc.go

```diff
@@ -5,13 +5,19 @@ import (
 	"sync"
 
 	"github.com/k3s-io/k3s/pkg/cli/cmds"
+	"github.com/k3s-io/k3s/pkg/util"
 	"github.com/sirupsen/logrus"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/tools/clientcmd"
 )
 
-const runtimeClassesChart = "rke2-runtimeclasses"
+const (
+	runtimeClassesChart  = "rke2-runtimeclasses"
+	namespace            = "kube-system"
+	helm                 = "Helm"
+	helmReleaseName      = "meta.helm.sh/release-name"
+	helmManageBy         = "app.kubernetes.io/managed-by"
+	helmReleaseNamespace = "meta.helm.sh/release-namespace"
```

Comment:
I would probably use the same consts as upstream: https://github.com/helm/helm/blob/main/pkg/action/validate.go#L33-L38
```suggestion
	appManagedByLabel              = "app.kubernetes.io/managed-by"
	appManagedByHelm               = "Helm"
	helmReleaseNameAnnotation      = "meta.helm.sh/release-name"
	helmReleaseNamespaceAnnotation = "meta.helm.sh/release-namespace"
```